src/main/java/com/synopsys/integration/blackduck/service/BlackDuckApiClient.java

```diff
@@ -165,6 +168,14 @@ public HttpUrl executePostRequestAndRetrieveURL(BlackDuckResponseRequest request
             throw new IntegrationException(e.getMessage(), e);
         }
     }
+    
+    public BlackDuckVersion getBlackDuckVersion() {
+		return blackDuckVersion;
+	}
```

Comment:
This should be made to return an Optional<BlackDuckVersion> otherwise we risk a NPE here.